src/common/BaseAnnotator.ts

```diff
@@ -76,10 +76,13 @@ export default class BaseAnnotator extends EventEmitter {
             api: new API({ apiHost, token }),
         });
 
+        this.handleChangeMode = this.handleChangeMode.bind(this);
```

Comment:
Let's continue to use arrow syntax for handlers rather than explicit `bind` calls.